src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -57,10 +57,15 @@ public ProcessSystem(World world, IParallelRunner runner) : base(world, runner,
     ///   </para>
     /// </remarks>
     public static void ComputeActiveProcessList(IReadOnlyList<IPositionedOrganelle> organelles,
-        [NotNull] ref List<TweakedProcess>? result)
+        [NotNull] ref List<TweakedProcess>? result, Dictionary<BioProcess, float> tempStorage)
     {
         result ??= new List<TweakedProcess>();
 
+        foreach (var process in result)
+        {
+            tempStorage.TryAdd(process.Process, process.SpeedMultiplier);
```

Comment:
For the hex work data the pattern I usually use is that the data is cleared just before use, this way if an exception breaks the game and skips the data clear, that won't cause compounding errors after that.